### PR TITLE
Chore: Disable/hide inapplicable network options in DAO Creation flow

### PR DESCRIPTION
**Pull request by RakeshUP**
Task: [APP-1394](https://aragonassociation.atlassian.net/browse/APP-1394)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

[APP-1394]: https://aragonassociation.atlassian.net/browse/APP-1394?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/web-app/src/containers/selectChainForm/index.tsx [hunk 1]
```diff
@@ -66,7 +64,8 @@ const SelectChainForm: React.FC = () => {
       <Header>
         <NetworkTypeSwitcher>
           <ButtonText
-            mode="secondary"
+            mode="ghost"
```
Comment (Fabricevladimir) [inline packages/web-app/src/containers/selectChainForm/index.tsx:66]: Can you do a quick update to the other button in the group as well? Both are ghost buttons with `bgwhite`. If you can get away with remove the "Sort by" label as well, that would be great 😄. I didn't even realize create DAO changed

File: packages/web-app/src/pages/createDAO.tsx [hunk 2]
```diff
@@ -112,7 +112,7 @@ const CreateDAO: React.FC = () => {
     // intentionally disabling this next line so that changing the
     // wallet network doesn't cause effect to run
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
+  }, [chainId]);
```
Comment (Fabricevladimir) [inline packages/web-app/src/pages/createDAO.tsx]: This is crucial. Including `chainId`  has the intended effect of overwriting previous choices. If the user comes in with their wallet connected to Goerli, Goerli will automatically be selected in the form. If they move on to the next step, and switch their network, this effect will run and set the chain to the new wallet chainId.

**Review by Fabricevladimir — COMMENTED** [review]
Thanks!

File: packages/web-app/src/pages/createDAO.tsx (hunk 2 above)
Comment (RakeshUP) [reply packages/web-app/src/pages/createDAO.tsx]: Thanks. Totally missed the comment above

**Review by Fabricevladimir — APPROVED** [review]

**Review by WissenIstNacht — APPROVED** [review]

**Pull request merged**